omnibus/files/server-ctl-cookbooks/infra-server/templates/default/redis_lb.conf.erb

```diff
@@ -35,3 +35,6 @@ activerehashing <%=@activerehashing%>
 
 auto-aof-rewrite-percentage <%=@aof_rewrite_percent%>
 auto-aof-rewrite-min-size <%=@aof_rewrite_min_size%>
+
+requirepass <%=@password%>
```

Comment:
Instead of leaving the plaintext password on disk,can you look at updating the recipe to start it, and if password isn't set issue a `CONFIG SET requirepass "#{PrivateChef.credentials.get('redis_lb', 'password')}"`?